From 998b8a6c8aff2f1364197abd0dc05ccb37e62801 Mon Sep 17 00:00:00 2001
From: Richard Fitzgerald <rf@opensource.cirrus.com>
Date: Wed, 22 Jul 2026 15:36:07 +0100
Subject: [PATCH 06493/15333] ASoC: cs35l56: Sort table of sdw_device_id

Swap the entries for 3562 and 3563 to keep the table in order of
increasing part number.

There's nothing broken here, it's just cosmetic.

Signed-off-by: Richard Fitzgerald <rf@opensource.cirrus.com>
Link: https://patch.msgid.link/20260722143607.1001473-1-rf@opensource.cirrus.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/codecs/cs35l56-sdw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/cs35l56-sdw.c b/sound/soc/codecs/cs35l56-sdw.c
index 1e442f43b306..303d37e7d0bf 100644
--- a/sound/soc/codecs/cs35l56-sdw.c
+++ b/sound/soc/codecs/cs35l56-sdw.c
@@ -510,8 +510,8 @@ static const struct dev_pm_ops cs35l56_sdw_pm = {
 static const struct sdw_device_id cs35l56_sdw_id[] = {
 	SDW_SLAVE_ENTRY(0x01FA, 0x3556, 0x3556),
 	SDW_SLAVE_ENTRY(0x01FA, 0x3557, 0x3557),
-	SDW_SLAVE_ENTRY(0x01FA, 0x3563, 0x3563),
 	SDW_SLAVE_ENTRY(0x01FA, 0x3562, 0x3562),
+	SDW_SLAVE_ENTRY(0x01FA, 0x3563, 0x3563),
 	{},
 };
 MODULE_DEVICE_TABLE(sdw, cs35l56_sdw_id);
-- 
2.55.0

